media/run-from-scratch.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+docker build -t orbitar-media . && \
```

Comment:
This is not needed in the final version, since the container should be run via docker-compose.